packages/contracts-bsc/contracts/BscTokenController.sol

```diff
@@ -187,6 +187,40 @@ contract BscTokenController {
         _;
     }
 
+    /**
+     * @dev sets the original `owner` of the contract to the sender
+     * at construction. Must then be reinitialized
+     */
+    constructor() public {
+        owner = msg.sender;
+        emit OwnershipTransferred(address(0), owner);
+    }
+
+    function initialize() public {
+        require(!initialized, "already initialized");
+        initialized = true;
+
+        owner = msg.sender;
+        emit OwnershipTransferred(address(0), owner);
+
+        instantMintThreshold = 150_000_000_000_000_000_000_000_000; // 150 M
```

Comment:
Reminder (from [this comment](https://github.com/trusttoken/contracts-pre22/pull/1233#discussion_r1139211653)) to make these human-legible numbers.

Reply:
Also, once you've manually granted `owner`ship to `msg.sender`, you should be able to call `setMintThresholds(_instant, _ratified, _multisig)` and `setMintLimits(_instant, _ratified, _multisig)` from the `initialize()` function, instead of duplicating code from elsewhere!

This may entail changing visibility of these two functions from `external` to `public`, but I think that's acceptable.